modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/healthcheck/LocalHealthcheck.scala

```diff
@@ -31,7 +31,10 @@ object LocalHealthcheck {
     mkPeersR.map(make(_, retryPolicy, nodeClient, clusterStorage))
   }
 
-  case class PeerUnresponsive(id: PeerId) extends NoStackTrace
+  case class PeerUnresponsive(id: PeerId) extends NoStackTrace {
+    implicit val show = PeerId.show
+    override val getMessage = id.show
```

Comment:
If we are adding the `getMessage` definition, maybe we should make it whole sentence? Like:
`Peer with id ${id.show} is unresponsive`

Reply:
That's what I would vote for regarding every message in this PR.

Reply:
@TheMMaciek I can do that; the reason I didn't is that the type name of the error already tells us that, and there's usually another message logged as well.